pkg/cmd/cli/cmd/process.go

```diff
@@ -89,7 +110,7 @@ Examples:
 			// If 'parameters' flag is set it does not do processing but only print
 			// the template parameters to console for inspection.
 			if cmdutil.GetFlagBool(cmd, "parameters") == true {
-				err = printer.PrintObj(templateObj, out)
+				err = printTemplateParameters(templateObj, out)
```

Comment:
Before, get on Template (printTemplate) called printTemplateParameters.  printTemplate now displays details of a template.

----- Original Message -----
> > @@ -89,7 +110,7 @@ Examples:
> >  			// If 'parameters' flag is set it does not do processing but only print
> >  			// the template parameters to console for inspection.
> >  			if cmdutil.GetFlagBool(cmd, "parameters") == true {
> > -				err = printer.PrintObj(templateObj, out)
> > +				err = printTemplateParameters(templateObj, out)
> 
> @mfojtik I don't think clayton was suggesting using printobj, just that you
> should pass the Parameters object, not the full template object.  you're not
> using any other fields of the object.
> 
> also who was calling printTemplateParameters before? I see you moved it, but
> no other code was changed...so was nothing calling it before?
> 
> 
> 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1218/files#r25795458